umap/static/umap/js/modules/rules.js

```diff
@@ -0,0 +1,240 @@
+import { DomUtil, DomEvent, stamp } from '../../vendors/leaflet/leaflet-src.esm.js'
+import * as Utils from './utils.js'
+import { translate } from './i18n.js'
+
+class Rule {
+  constructor(map, condition = '', options = {}) {
+    this.map = map
```

Comment:
If these are private, should we prefix them with `#`?

Reply:
Sadly, the browser support is not enough for us for this feature yet :(